app/src/views/GrantRoundDetails.vue

```diff
@@ -48,11 +50,12 @@
         Address:
         <a class="link" :href="`https://etherscan.io/address/${grantRound.address}`">{{ grantRound.address }}</a>
       </p>
-      <p>
-        Metadata URL: <a class="link" :href="grantRound.metaPtr" target="_blank">{{ grantRound.metaPtr }}</a>
-      </p>
 
       <button @click="startContributing" class="btn btn-primary mt-6">Contribute to the matching fund</button>
+
+      <!-- Grants Belonging to Grants Round -->
+      <!-- TODO: isGrantRoundView=true -->
+      <GrantRegistryList />
```

Comment:
Q: 

- Right now we have a single `poll` function that fetches all grants and grants Rounds. 
With this PR when the user visits the grant round page -> they end up being shown a subset of the grants as opposed to all the grants. I'm not clear on how / where to do the filtering so that all grants aren't shown. I'm trying to avoid writing a duplicate of GrantRegistryList as only the data needs to be updated

- I need to pass isGrantRoundView so that the title is hidden when GrantRegistryList is rendered but I'm not sure why I am not able to do that

Reply:
If I'm understanding the `poll` code correctly, it currently fetches every grant in the registry and the round data from on chain. We need to also fetch the round metadata. Since we'll then have all the data required, the filtering happens on the round page. Looking at the `grants` field after the GrantsRound is returned, we should have a list of ids like `[1, 2, 4, 10]`. So the Round page should filter the global registry grant list down to only those found in the `grants` field, and simply show those.

As a side note, I don't think this architecture will work long term, as the number of grants will be an impediment to this zealous up-front fetching. But it should be ok for the mvp round.

Reply:
I think there are a few options, depending on whether we still want to keep a `GrantRegistryList` page that lists all grants, regardless of the round the grant is in. For all of them, we'll want to move the `GrantRegistryList.vue` file from `/views` (i.e. pages) to `/components` (i.e. reusable components) and possibly rename it. Then either
1. Change the `grants` variable from something that is read from the data store to a prop that is passed in
2. Add a `filterFn` prop that takes a method used to filter the `grants` data from the list of all grants to some subset, i.e. grants by round, and in the `setup()` method return the filtered grants list (probably as a computed property) instead of just returning grants as-is

> As a side note, I don't think this architecture will work long term, as the number of grants will be an impediment to this zealous up-front fetching. But it should be ok for the mvp round.

Agreed, we have various parts that won't scale well, often with `TODO` comments. Probably should create a "scaling" meta-issue at some point to track these